qutebrowser/mainwindow/tabwidget.py

```diff
@@ -705,7 +705,17 @@ def wheelEvent(self, e):
             e: The QWheelEvent
         """
         if config.val.tabs.mousewheel_switching:
-            super().wheelEvent(e)
+            if utils.is_mac:
+                index = self.currentIndex()
+                if index != -1:
+                    if e.angleDelta().y() > 0:
+                        index += -1
+                    else:
+                        index += 1
```

Comment:
This could be

```suggestion
                    offset = -1 if e.angleDelta().y() > 0 else 1
                    index += offset
```

to make it (IMHO) a bit easier to read. I'm okay with both versions though. If you decide to keep yours, use `-= 1` instead of `+= -1` though.